repos/spack_repo/builtin/packages/motioncor2/package.py

```diff
@@ -19,46 +17,76 @@ class Motioncor2(Package):
     to keep up with automated data collection."""
 
     homepage = "http://msg.ucsf.edu/em/software"
-    manual_download = True
+    # Use Scipion's download server as the official site no longer hosts some deprecated versions
+    url = "https://scipion.cnb.csic.es/downloads/scipion/software/em/motioncor2-1.6.4.tgz"
 
-    version("1.6.4", sha256="28bb3e6477abf34fe41a78bcb9da9d77d08e2e89ecd41240fab085a308e6c498")
-    version("1.4.7", sha256="8c33969b10916835b55f14f3c370f67ebe5c4b2a9df9ec487c5251710f038e6b")
+    version("1.6.4", sha256="bcb70ddde38daa7722e007f2509bae928c8603530524b6a36ed79f4ca7b3edb8")
+    version("1.5.0", sha256="20cdb46efe3f3c578071ddbd9e9f45c8462b0cbb2fc5b3cfb2d0b66ded0d30e0")
+    version("1.4.5", sha256="2f04bceb6069cd4f91a51447aac07b887be46206e094555bc445e89944ed2851")
 
-    # None of the below are available for download
-    version(
-        "1.1.0",
-        sha256="6e37e7ed63a9f0aab5d794b2604d5ba79333960bb9440a1a218630b03dbeaeac",
-        deprecated=True,
-    )
-    version(
-        "1.0.5",
-        sha256="4efa55af25644bcff1ca7882419267b8c094c9cc6155b37d2c204b154c56f5a8",
-        url="http://msg.ucsf.edu/MotionCor2/MotionCor2-1.0.5.tar.gz",
-        deprecated=True,
-    )
-    version(
-        "1.0.4",
-        sha256="c75738160ac18d3f27c33677e78e63313d8ec2b023b5a46173428c3fa0451a94",
-        url="http://msg.ucsf.edu/MotionCor2/MotionCor2-1.0.4.tar.gz",
-        deprecated=True,
-    )
+    # Deprecated versions, no longer available in official site
+    with default_args(deprecated=True):
+        version("1.6.3", sha256="16c97516992d638ea817a943185213874dfad7d87a6bb71e102598922bb35c75")
+        version("1.4.7", sha256="1c6ee6b76413db2d044a0048957a23907ad3217225d5e1a3659df3c3947d0e2c")
+        version("1.4.4", sha256="1ced27bbc46a437ea1e5af8c9c842c9789da100d860d3663fc8215266cb4678e")
+        version("1.4.2", sha256="97735aa22d01135933c5b698d8990ed371a89aec90df5193a24400c6460eb276")
+        version("1.4.0", sha256="d68c4ab9c6a740a73aede7120854a1fb3aea94279afb5fe2ab50387b725e6329")
+        version("1.3.2", sha256="d9fb3537278be179fd9bfa45f58e921649a893b169cbf8fa173ba95fdb831d10")
+        version("1.3.1", sha256="861fb9ab9fbe0ca72e538ef7c2f8e5d2ebd02d9b6c852c8d57a48782a3a89547")
+        version("1.3.0", sha256="7da654830ec64c4b66a965d184d008f310d90fa4594b9166da1d530381275e5f")
+        version("1.2.6", sha256="b16d512bd4a122563d73a568d44bb724fb8b96d0d20662ce1a94274fb39dfaa6")
+        version("1.2.3", sha256="49dfe25316f9f65f77861383a583e1cd9b466abd6dc0e9f446ffbdefdafeb370")
+        version("1.2.1", sha256="b68db6850d64fc56635ff968b53587c96bb42108ba58a415bef447f0810b9e57")
+        version("1.1.0", sha256="1cb7c486d7f575fefcc7dae5037e9bb645842e1e43b22fa3c18fd0e4d2a189db")
```

Comment:
> MotionCor2 is known to make breaking changes in patch versions so I'd rather keep these deprecated versions just in case someone needs them.

I think what alalazo said would still holds, because I don't see that these versions existed before:

```suggestion
```